_posts/blog/2019-09-19-2019-09-19-lab-meeting.md

```diff
@@ -0,0 +1,105 @@
+She took a couple of pictures of Kirstie hosting the [Early Career Investigator Data Blitz](https://github.com/WhitakerLab/DataBlitzResources) and particularly enjoyed Dr Lola Cañamer’s presentation on [Emotion Modeling in Robots](http://emotion-modeling.info/)!
+Cañamer and her lab made some affective autonomous and social robots with the goal of understanding emotions in biological systems (humans and other animals) and how to build a better robot.
+
+<figure class="half">
+  <img src="/images/lab-meeting/2019-09-19/abc2019-datablitz-kirstie.jpg" alt="Kirstie on stage introducing data blitz">
+  <img src="/images/lab-meeting/2019-09-19/abc2019-emotionalrobots.jpg" alt="Slide of talk about embodied affect in robots">
+  <figcaption>Pictures from Cambridge Neuroscience Symposium #ABC2019.
+              Taken by Yini He, 12 & 13 September 2019.
+  </figcaption>
+</figure>
+
+**Georgia** gave Patricia, Ang, Yini and Kirstie a demo of the Open Science Game she and her team created at the [eLife Innovation Sprint](https://sprint.elifesciences.org).
+We loved it, and you can check out the project on GitHub at [https://github.com/GeorgiaHCA/OpenScienceGame](https://github.com/GeorgiaHCA/OpenScienceGame).
+
+<figure>
+  <center>
+  <img src="/images/lab-meeting/2019-09-19/open-science-game-twitter.png" alt="Screen shot of tweet with pictures of team members playing card game" width="70%">
+  <figcaption>Playing a game to explore the incentives and challenges of open research practices!<br>
+              Source: <a href="https://twitter.com/kirstie_j/status/1174030821193392129?s=20">Kirstie's twitter feed</a>
+  </figcaption>
+  </center>
+</figure>
+
+## Questions we're thinking about
+
+**Kirstie** is getting excited for Mozilla's annual festival: [www.mozillafestival.org](https://www.mozillafestival.org/en/) on 21 to 27 October.
+She didn't put in any session applications this year but would love for folks in the group to share what they're working on as part of the many many sessions on [openness](https://www.mozillafestival.org/en/spaces/openness/), [digital inclusion](https://www.mozillafestival.org/en/spaces/digital-inclusion/), [neurodiversity](https://www.mozillafestival.org/en/spaces/neurodiversity/) and [web literacy](https://www.mozillafestival.org/en/spaces/web-literacy/) 💖
+
+You can also support **Yo**'s sessions on ["Exhausted impostors and superstar masks"](https://public.zenkit.com/i/2RH604FcHf/piV4I8VtX/exhausted-impostors-and-superstar-masks?v=A2QcaWQ-QO) (with [Jez Cope](https://twitter.com/jezcope) and [Emmy Tsang](https://twitter.com/emmy_ft), and inspired by a [twitter thread](https://twitter.com/yoyehudi/status/1118474644766560256))and on ["Choosing an effective mentorship program"](https://public.zenkit.com/i/2RH604FcHf/sCleORy_8MO/choosing-an-effective-mentorship-program?v=A2QcaWQ-QO) (with **Malvika**).
```

Comment:
```suggestion
You can also support **Yo**'s sessions on ["Exhausted impostors and superstar masks"](https://public.zenkit.com/i/2RH604FcHf/piV4I8VtX/exhausted-impostors-and-superstar-masks?v=A2QcaWQ-QO) (with [Jez Cope](https://twitter.com/jezcope) and [Emmy Tsang](https://twitter.com/emmy_ft), and inspired by a [twitter thread](https://twitter.com/yoyehudi/status/1118474644766560256)) and on ["Choosing an effective mentorship program"](https://public.zenkit.com/i/2RH604FcHf/sCleORy_8MO/choosing-an-effective-mentorship-program?v=A2QcaWQ-QO) (with **Malvika**).
```

Reply:
The change isn't showing clearly on here, but it's the tiniest of tiny edits! Needed a space between "twitter thread" and "and".

Reply:
You're the best @LouiseABowler - thank you!